TestValidateTimeout fails on go v1.15.6

```bash
➜ go test -v
=== RUN   TestAddGet
--- PASS: TestAddGet (0.00s)
=== RUN   TestPrune
--- PASS: TestPrune (3.00s)
=== RUN   TestProbeHandler
--- PASS: TestProbeHandler (0.00s)
=== RUN   TestGetTimeout
--- PASS: TestGetTimeout (0.00s)
=== RUN   TestCreateLogger
--- PASS: TestCreateLogger (0.00s)
=== RUN   TestValidateTimeout
    ssllabs_exporter_test.go:139: Test case : un_parsable_duration failed.
        Expected : (0s, time: invalid duration not_a_duration)
        Got : (0s, time: invalid duration "not_a_duration")
--- FAIL: TestValidateTimeout (0.00s)
FAIL
exit status 1
FAIL    github.com/anas-aso/ssllabs_exporter    3.006s
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -11,7 +11,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v1
       with:
-        go-version: 1.13
+        go-version: 1.15
     - name: Checkout code
       uses: actions/checkout@v1
     - name: Cache dependencies

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,8 +1,6 @@
 name: test
 on:
   push:
-    branches:
-      - master
   pull_request:
 jobs:
   test:

--- a/pkg/ssllabs/info_test.go
+++ b/pkg/ssllabs/info_test.go
@@ -26,7 +26,7 @@ func TestInfo(t *testing.T) {
 	}
 
 	expectedInfo := APIInfo{
-		EngineVersion:      "2.1.3",
+		EngineVersion:      "2.1.8",
 		CriteriaVersion:    "2009q",
 		MaxAssessments:     25,
 		CurrentAssessments: 0,

--- a/ssllabs_exporter_test.go
+++ b/ssllabs_exporter_test.go
@@ -117,7 +117,7 @@ func TestValidateTimeout(t *testing.T) {
 			name:            "un_parsable_duration",
 			flagTimeout:     "not_a_duration",
 			expectedTimeout: 0,
-			expectedError:   errors.New("time: invalid duration not_a_duration"),
+			expectedError:   errors.New(`time: invalid duration "not_a_duration"`),
 		},
 		{
 			name:            "less_than_1m",

